docs/01-start-here/04-troubleshooting.md

```diff
@@ -2,6 +2,33 @@
 
 ## Frequent Issues
 
+### Start clean
+
+Sometimes, especially if you've not worked on Frontend for a while, things can get out of sync and it's worth have a spring clean.
+
+You can do this by running `clean` in your project folders:
+
+```
+./sbt
+[root] project dev-build
+[dev-build] clean
+```
+
+And then running `compile`:
+
+```
+[dev-build] compile
+```
+
+If you're still seeing errors, try clearing out your target folders:
```

Comment:
Maybe before this, we could recommend running the mighty `cleanAll` command at `root`?

Reply:
Not sure, I thought `cleanAll` was more all-encompassing - does it clean all projects? Might be worth asking a server side person 😝

Reply:
`git clean -fxd` is the strongest cleaner here

Reply:
Re the cleanAll stuff, yep it cleans all of the Scala SBT projects. The cost is that you have to recompile everything/whatever you are working on fully next time you run it (can be up to 10 mins for dev-build) but sometimes it's needed!

Reply:
Okay, given that following a cleanAll and deleting targets the application takes an age to recompile, I'd say this is the order we should try things in:

- `clean` for individual project
- `cleanAll` at root
- `git clean -fxd` (deletes all ignored code, i.e. `target/` directories, etc?)

Does everyone agree?

Reply:
Agree.

Have updated.